From: Debajyoti Bera <dbera.web@gmail.com>

Assume local time for exif timestamps. Its what the camera clock says, and I doubt sane people put UTC on clocks.

Index: Exif.cs
===================================================================
--- Exif.cs	(revision 3413)
+++ Exif.cs	(working copy)
@@ -273,7 +273,8 @@
 					Int32.Parse(dt_data[2]),
 					Int32.Parse(dt_data[3]),
 					Int32.Parse(dt_data[4]),
-					Int32.Parse(dt_data[5]));
+					Int32.Parse(dt_data[5]),
+					DateTimeKind.Local); // Assume local time
 			} catch (FormatException) {
 				result = DateTime.MinValue;
 			}
